### PR TITLE
webui: update localization

### PR DESCRIPTION
**Pull request by fbergkemper**

### CODE REVIEW

**Comment by arogge** [comment]
There are a lot of empty files in that commit. Is that correct.

**Comment by fbergkemper** [comment]
> There are a lot of empty files in that commit. Is that correct.

Oops, something went wrong. Should be fixed now.

File: webui/scripts/localization_update.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ echo 'Message lookup ...'
 
 find ../ -regextype posix-egrep -regex '.*(php|phtml)$$' | grep -v vendor | grep -v tests | xargs xgettext --keyword=translate -L PHP --from-code=UTF-8 -o $POTFILE;
 find ../ -regextype posix-egrep -regex '.*(functions.js)$$' | xargs xgettext --keyword=gettext --from-code=UTF-8 -j -o $POTFILE;
+find ../ -regextype posix-egrep -regex '.*(formatter.js)$$' | xargs xgettext --keyword=gettext --from-code=UTF-8 -j -o $POTFILE;
```
Comment (arogge) [inline webui/scripts/localization_update.sh]:
```suggestion
find ../ -regextype posix-egrep -regex '.*(functions.js|formatter.js)$$' | xargs xgettext --keyword=gettext --from-code=UTF-8 -j -o $POTFILE;
```

Comment (arogge) [reply webui/scripts/localization_update.sh]: The two lines can be joined together to avoid duplication

Comment (fbergkemper) [reply webui/scripts/localization_update.sh]: that true

**Pull request merged**